setup.cfg

```diff
@@ -15,9 +15,6 @@ classifiers =
     License :: OSI Approved :: MIT License
     Programming Language :: Python :: 3
     Programming Language :: Python :: 3 :: Only
-    Programming Language :: Python :: 3.8
```

Comment:
Removed by the `setup-cfg-fmt`. If we need to keep them we need to provide an extra option, per:

https://github.com/asottile/setup-cfg-fmt#adds-python-version-classifiers